groovy/gradle.java/nbproject/project.properties

```diff
@@ -20,7 +20,10 @@ javac.source=1.8
 javac.compilerargs=-Xlint -Xlint:-serial
 nbm.module.author=Laszlo Kishalmi
 
+test-unit-sys-prop.test.netbeans.dest.dir=${netbeans.dest.dir}
```

Comment:
See comment in the testcase.

Reply:
This one is needed to get the installed file locator working properly. Without that the Gradle side plugin cannot be loaded.